src/test/java/cloud/eppo/helpers/BanditTestCase.java

```diff
@@ -26,11 +41,77 @@ public List<BanditSubjectAssignment> getSubjects() {
     return subjects;
   }
 
-  public void setFileName(String fileName) {
-    this.fileName = fileName;
+  public static Stream<Arguments> getBanditTestData() {
```

Comment:
And here is why I relocated all the `BanditTestCase`-related utility methods, also public.